sparkplug-core/src/java/sparkplug/function/SerializableFn.java

```diff
@@ -60,6 +64,72 @@ protected SerializableFn(IFn fn, Collection<String> namespaces) {
     }
 
 
+    /**
+     * Safely access the value of a field on the given object.
+     *
+     * @param obj Instance to access a field on
+     * @param field Reflective field to access
+     * @return the value of the field, or nil on failure
+     */
+    public static Object accessField(Object obj, Field field) {
+        try {
+            if (!field.isAccessible()) {
+                field.setAccessible(true);
+            }
+            return field.get(obj);
+        } catch (Exception ex) {
+            logger.trace("Failed to access field " + field.toString() + ": " + ex.getClass().getName());
+            return null;
+        }
+    }
```

Comment:
Does the Java version of `accessField` behave differently than the prior Clojure version?

Reply:
The logic is identical, except that the Java version uses the SLF4J logging interface, and the Clojure version was using `clojure.tools.logging` (which ultimately forwarded to the SLF4J backend).